### DISCUSSION TITLE
German Translation completed

### DISCUSSION THREAD

**Discussion by omtr** [SiimKirjanen/seatreg#101] [General]
Hi Siim,

i just completed the German translation on https://translate.wordpress.org/projects/wp-plugins/seatreg/stable/de/default/ .
Can you give me directions how i can use them?

Thank you!

**Comment by SiimKirjanen** [comment]
Hey. Thank you for the translations 👍 
Next step is to approve those translations so people can use them. We need someone who has permissions to approve German translations.
I don't have that permission sadly. There are some translation editors mentioned in: https://translate.wordpress.org/locale/de/default/wp-plugins/seatreg/
I think there is also possibility for you to become the translation editor for this plugin. It has happened before: https://make.wordpress.org/polyglots/2021/10/03/seatreg-translated-to-danish-howto/

**Comment by omtr** [comment]
done! i requested PTE for the german locale.
https://make.wordpress.org/polyglots/?p=47243

**Comment by fc-steinfeld** [comment]
Hey everyone,

I updated the translations, where changes were requested, provided the missing ones and got them reviewed. We are now at 84% completion. What needs to happen to get them into the plugin?
https://translate.wordpress.org/projects/wp-plugins/seatreg/

@SiimKirjanen can you create a new version, to include them, or what is the process here?

I'd love for this to happen this week, if possible, because we are excited to use the plugin this Saturday for the first time live on our website 🎉

Let me know if I can do anything to support this.

**Comment by SiimKirjanen** [comment]
Hey,

Thank you for the translations. We need to get 90% of translations completed. After that the language pack will be generated
https://make.wordpress.org/polyglots/handbook/frequently-asked-questions/#strings-are-validated-when-are-they-taken-into-account-for-download

**Comment by fc-steinfeld** [comment]
Done, thanks a lot!
